tests/fsharp/tests.fs

```diff
@@ -28,6 +28,84 @@ let FSI_BASIC = FSI_FILE
 // ^^^^^^^^^^^^ To run these tests in F# Interactive , 'build net40', then send this chunk, then evaluate body of a test ^^^^^^^^^^^^
 
 module CoreTests = 
+    open TestFramework
+
+#if !FSHARP_SUITE_DRIVES_CORECLR_TESTS
+    [<Test>]
+    let testFSharpPlusBuild () =
+        let cfg = testConfig ""
+        mkdir cfg "repos"
+        
+        if Commands.directoryExists cfg.Directory "repos/FSharpPlus" |> Option.isNone then
+            let cfg = testConfig "repos"
+            git cfg ["clone"; "http://github.com/fsprojects/FSharpPlus"; "FSharpPlus"]
+
+        let cfg2 = testConfig "repos/FSharpPlus"
+
+        git cfg2 ["reset"; "--hard"; "aba256430a86a4022941800f06421dda16aa4cb0"]
```

Comment:
You can use the `--branch` argument to `git clone` to clone at a specific commit